Polyhedron/demo/Polyhedron/Plugins/IO/XYZ_io_plugin.cpp

```diff
@@ -73,9 +73,6 @@ class Polyhedron_demo_xyz_plugin :
   bool canSave(const CGAL::Three::Scene_item*);
   bool save(const CGAL::Three::Scene_item*, QFileInfo fileinfo);
   bool applicable(QAction*) const { return true;}
-  QList<QAction*> actions() const {
```

Comment:
this function is missing now, and causes a compilation error : "cannot instantiate abstract class" for `Polyhedron_demo_xyz_plugin`